### DISCUSSION TITLE
Example Configurations

### DISCUSSION THREAD

**Discussion by pertrai1** [helix-editor/helix#3919] [Q&A] [answered]
I am trying this out and was wondering if there are examples I can get inspiration from? Maybe a list of user dotfiles that are using helix? I always love look at other peoples vim configurations as I get inspiration from those configs. I would be interested in doing the same with helix. I did a quick search to see if this has already been asked for. If it has and I overlooked it, I apologize for not investigating further. Thanks.

**Comment by the-mikedavis** [comment, marked as answer]
There are not many configuration options overall: https://docs.helix-editor.com/configuration.html and many configs take fewer than 10 lines total. There are a lot of features that are built-in and enabled by default so you shouldn't need config starter packs like you usually would need for vim/emacs/...

**Reply by pertrai1** [reply]
Thanks for the reply. I look forward to playing around with helix.

**Comment by kirawi** [comment]
See #1360
